### PR TITLE
AVM Vertex serialization specification

### PR DESCRIPTION
**Pull request by cgcardona**
* Vertex
* What Vertex Contains
* Gantt Vertex Specification
* Proto Vertex Specification
* Vertex Example

### CODE REVIEW

File: build/avalanchego-apis/index-api.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ This document shows how to query data from AvalancheGo's Index API. The Index AP
 
 This API uses the `json 2.0` RPC format. For more information on making JSON RPC calls, see [here](issuing-api-calls.md).
 
+For data format, please refer to following:
+* [AVM (X-Chain) Transaction Format](../references/avm-transaction-serialization.md)
+* [Platform Transaction Format](../references/platform-transaction-serialization.md)
+* [Coreth Atomic Transaction Format](../references/coreth-atomic-transaction-serialization.md)
+
```
Comment (yulin-dong) [inline build/avalanchego-apis/index-api.md:19]: Added this section for cross-referencing. Please check if they are correct @cgcardona @danlaine

Comment (cgcardona) [reply build/avalanchego-apis/index-api.md:19]: Looks good. Thank you @yulin-dong! 🎩

File: build/avalanchego-apis/index-api.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
+# Index API
```
Comment (yulin-dong) [inline build/avalanchego-apis/index-api.md:1]: This file has been moved to a new location on /apis/avalanchego/apis/index-api. Please update that doc accordingly.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -15,38 +17,41 @@ The website is built using [Docusaurus 2](https://docusaurus.io/).
...
 ```
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
 
 ### Build
 
-```
-$ yarn build
+```zsh
+> yarn build
```
Comment (yulin-dong) [inline README.md:48]: remove > or add it to all other places.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**